src/promptflow/promptflow/_sdk/_utils.py

```diff
@@ -897,3 +904,34 @@ def dump_flow_result(flow_folder, prefix, flow_result=None, node_result=None):
     if output:
         with open(dump_folder / f"{prefix}.output.json", "w") as f:
             json.dump(output, f, indent=2)
+
+
+def _resolve_base64_image_in_test_result(test_result, flow_folder, sub_dir):
```

Comment:
Is this PR a ask from VSC?

Reply:
Yes, this ark is related to VSC, reported by Heyi. After changing to the path format, customer can jump directly to the image file in VSC
![image](https://github.com/microsoft/promptflow/assets/17938940/5335aa4a-6bca-41cc-b837-2c690a5af6b5)